administrator/language/de-DE/plg_fields_radio.ini

```
@@ -6,6 +6,8 @@
 
 PLG_FIELDS_RADIO="Felder - Optionsfeld"
 PLG_FIELDS_RADIO_LABEL="Optionsfeld (%s)"
+PLG_FIELDS_RADIO_PARAMS_FORM_LAYOUT_BUTTONS="Buttons"
+PLG_FIELDS_RADIO_PARAMS_FORM_LAYOUT_SWITCHER="Switcher"
```

```suggestion
PLG_FIELDS_RADIO_PARAMS_FORM_LAYOUT_SWITCHER="Umschalter"
```

@zero-24 was meinst?

---

```suggestion
PLG_FIELDS_RADIO_PARAMS_FORM_LAYOUT_SWITCHER="Schalter"
```

Wäre Schalter nicht noch besser?